homework/shelkovich_ilya/enumCalculator/src/main/java/com/tutrit/java/quickstart/bean/Operations.java

```diff
@@ -0,0 +1,29 @@
+package com.tutrit.java.quickstart.bean;
+
+import com.tutrit.java.quickstart.exceptions.CalculatorException;
+import com.tutrit.java.quickstart.exceptions.ZeroException;
+
+public enum Operations {
+    PLUS,
+    MINUS,
+    MULTI,
```

Comment:
MULTY

Reply:
in this case is MULTI because its multiplication